plugins/rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/vuln_search/action.py

```diff
@@ -0,0 +1,36 @@
+import insightconnect_plugin_runtime
+from .schema import VulnSearchInput, VulnSearchOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+class VulnSearch(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="vuln_search", description=Component.DESCRIPTION, input=VulnSearchInput(), output=VulnSearchOutput()
+        )
+
+    def run(self, params={}):
+        asset_crit = params.get(Input.ASSET_CRITERIA)
+        vuln_crit = params.get(Input.VULN_CRITERIA)
+        size = params.get(Input.SIZE, 200)
+        sort_criteria = params.get(Input.SORT_CRITERIA, dict())
+        parameters = list()
```

Comment:
That's fine 😄 I prefer to write it more like in this way:
```suggestion
        sort_criteria = params.get(Input.SORT_CRITERIA, {})
        parameters = []
```
But it's up to you 😄